README.md

```diff
@@ -93,7 +93,7 @@ To contribute code, fork the repository, apply changes and submit a pull request
 
 Requirements:
 
-- Node.js v16 LTS
+- Node.js v18 LTS
```

Comment:
Should this be `Node.js LTS version, v18 recommended`? As v18 isn't a hard requirement

Reply:
Good point 👍

Reply:
Sources are updated